core/store/src/db.rs

```diff
@@ -777,10 +777,10 @@ mod tests {
     impl RocksDB {
         #[cfg(not(feature = "single_thread_rocksdb"))]
         fn compact(&self, col: DBCol) {
-            self.db.compact_range_cf::<&[u8], &[u8]>(
+            self.db.compact_range_cf(
                 unsafe { &*self.cfs[col as usize] },
-                None,
-                None,
+                Option::<&[u8]>::None,
```

Comment:
Why this change?

Reply:
The type of the argument changed from `&[u8]` to `Option<AsRef<[u8]>>` and with that the compiler could no longer deduce type.